### PR TITLE
Auto TC OCP-37479

### PR DESCRIPTION
**Pull request by chao007**
@jhou1 @duanwei33 Please help review resizing volume test case.
Have passed in local. But did not run from Jenkins now due to Jenkins issue today.

### CODE REVIEW

File: features/storage/csi_resize.feature [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    """
+    When I execute on the pod:
+      | /bin/dd | if=/dev/zero | of=/mnt/iaas/1 | bs=1M | count=1500 |
+    Then the step should succeed
+    And the output should not contain:
+      | No space left on device |
+
+    Examples:
+      | sc_name |
+      | standard-csi | # @case_id OCP-37479
```
Comment (jhou1) [inline features/storage/csi_resize.feature:45]: Align the tables `|`

File: features/storage/csi_resize.feature [hunk 2]
```diff
@@ -0,0 +1,45 @@
+Feature: CSI Resizing related feature
+  # @author chaoyang@redhat.com
+  @admin
+  Scenario Outline: Resize online volume from 1Gi to 2Gi
+    Given I have a project
+    And admin clones storage class "sc-<%= project.name %>" from "<sc_name>" with:
+      | ["metadata"]["name"] | sc-<%= project.name %>      |
```
Comment (jhou1) [inline features/storage/csi_resize.feature:7]: Redundant space after `>`

File: features/storage/csi_resize.feature [hunk 3]
```diff
@@ -0,0 +1,45 @@
+Feature: CSI Resizing related feature
+  # @author chaoyang@redhat.com
+  @admin
+  Scenario Outline: Resize online volume from 1Gi to 2Gi
```
Comment (jhou1) [inline features/storage/csi_resize.feature:4]: Do you mind rephrase this scenario? What does `online volume` mean?
For example:`Increase the volume size`

Comment (duanwei33) [reply features/storage/csi_resize.feature:4]: /lgtm

Comment (chao007) [reply features/storage/csi_resize.feature:4]: `online`  means that volume is in-used by a pod. We don't need to restart the pod when doing the volume expansion.

File: features/storage/csi_resize.feature [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    Given I have a project
+    And admin clones storage class "sc-<%= project.name %>" from "<sc_name>" with:
+      | ["metadata"]["name"] | sc-<%= project.name %> |
+    Then the step should succeed
```
Comment (liangxia) [inline features/storage/csi_resize.feature:8]: We do not overwrite any attributes for the storage class ?   Sound like we do not need the clone storage class there then.

Comment (chao007) [reply features/storage/csi_resize.feature]: @liangxia Thanks for the comments. Have updated.

**Comment by chao007** [comment]
@liangxia Could you help to review it?

**Comment by jhou1** [comment]
LGTM.

Please rebase and squash these commits.

**Pull request merged**